app/javascript/components/server-components/BalancePage.tsx

```diff
@@ -598,14 +600,266 @@ const PeriodPaypalAccount = ({
 
 const PeriodNoAccount = () => <h4>Will be sent to:</h4>;
 
+type InstantPayout = {
+  payable_amount_cents: number;
+  payable_balances: {
+    id: string;
+    date: string;
+    amount_cents: number;
+  }[];
+  bank_account_type: string;
+  bank_name: string | null;
+  routing_number: string;
+  account_number: string;
+};
+
+const InstantPayoutsNotice = () => (
```

Comment:
There are no changes here, I just created a separate component for markup that was previously inline.